Topics/Development_Process/Build_Requirements/Introduction_to_make_and_Makefiles.md

```diff
@@ -0,0 +1,228 @@
+# Introduction to ```make``` and Makefiles
+
+## Table of contents
+
+### [Introduction](#introduction-1)
+
+### [Basic Makefile Characteristics](#basic-makefile-characteristics-1)
+
+### [Making a Makefile](#making-a-makefile-1)
+
+### [Advantages and Disadvantages to ```make``` and Makefiles](#advantages-and-disadvantages-to-make-and-makefiles-1)
+
+### [Additional Resources](#additional-resources-1)
+
+## Introduction
+
```

Comment:
I think it would be nice to add that this guide requires some prerequisite C compilation and Unix shell knowledge to understand it.

Reply:
Adding some resources for these topics would help too!

Reply:
Those would be pretty useful, thanks for the suggestion.